GenderPayGap.WebUI/Views/Viewing/EmployerDetails/BonusPay/BonusPaySection.cshtml

```diff
@@ -58,15 +58,13 @@
         <div class="grid-row">
             <div class="column-two-thirds">
                 <span>
-                    In this organisation, <strong>women earn @(Model.FemaleMedianBonusPayMonitised)</strong>
-                    for every <strong>@(Model.MaleMedianBonusPayMonitised)</strong>
+                    In this organisation, women earn @(Model.FemaleMedianBonusPayMonitised)

```

Comment:
Surprised we aren't changing this to something like 'At this employer' or something similar, I guess we just leave it for now.